include/eve/detail/category.hpp

```diff
@@ -133,4 +133,13 @@ namespace eve::detail
       return category::invalid;
     }
   }
+
+  template<typename W, typename V, typename... Cat>
+  EVE_FORCEINLINE consteval bool
+  have_common_category(Cat... tst)
+  {
+    auto w_cat = categorize<W>();
+    auto v_cat = categorize<V>();
+    return ((match(w_cat, tst) && match(v_cat, tst)) || ...);
+  }
```

Comment:
You have one usage. Just inline. It's not a good name.

Reply:
Removed, as it seems `std::bit_cast` works fine